packages/server/database/migrations/20221025161717-addIsPremiumMeetingTemplate.ts

```diff
@@ -0,0 +1,26 @@
+import {R} from 'rethinkdb-ts'
+
+const premiumTemplateIds = [
+  'startStopContinueTemplate',
+  'gladSadMadTemplate',
+  'whatWentWellTemplate',
+  'workingStuckTemplate',
+  'sailboatTemplate',
+  'original4Template',
+  'fourLsTemplate'
+]
+
+export const up = async function (r: R) {
+  await r
+    .table('MeetingTemplate')
+    .getAll(r.args(premiumTemplateIds))
+    .update({isPremium: true})
```

Comment:
I thought these 7 templates should be FREE? i.e., not premium? Did I interpret the requirement wrong?

Reply:
You're completely right! 🤦‍♂️ I've changed it to `isFree` instead of `isPremium` as I think it makes the logic easier to read